server/connection.js

```diff
@@ -0,0 +1,13 @@
+const mysql = require('mysql');
+const mysqlConfig = require('../config.js');
```

Comment:
resolve: {
    extensions: ['.js', '.jsx']
  },

If you add this to your webpack config file after the rules part, you can take out the .js and .jsx part when importing files.